plugins/plugin-site/src/templates/search.jsx

```diff
@@ -61,7 +61,6 @@ const doSearch = (data, setResults, categoriesMap) => {
             plugins: hits
         });
     }).catch(err => {
-        window?.Sentry?.captureException(err);
         // FIXME alert/console.log/somehow tell user something went wrong
```

Comment:
Should this be revised now that Sentry will be removed?

Reply:
We can remove the catch call if that's okay

Reply:
Removed it

Reply:
That wasn't what I meant ^^
With Sentry removed I think there should be *at least* a console.log

Reply:
Got it. Updated it accordingly.